CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+* Bump Rubocop to version 0.64 to fix Ruby 2.6.1 issue
+* Bump Ruby to 2.6.1
```

Comment:
This entry is a bit unnecessary, as the `.ruby-version` is just an internal thing.